bfe_basic/condition/primitive.go

```diff
@@ -1043,14 +1043,14 @@ func (f *BfeTimeFetcher) Fetch(req *bfe_basic.Request) (interface{}, error) {
 	return debugTime, nil
 }
 
-// periodic time range matcher
+// PeriodicTimeMatcher periodic time range matcher
 type PeriodicTimeMatcher struct {
 	startTime int // in seconds of a day
 	endTime   int
 	offset    int // timezone offset
 }
 
-// time string format: hhmmssZ, example 150405H, Z-> timezone defined in bfe_util.TimeZoneMap
+// NewPeriodicTimeMatcher time string format: hhmmssZ, example 150405H, Z-> timezone defined in bfe_util.TimeZoneMap
```

Comment:
// NewPeriodicTimeMatcher follows time string format: hhmmssZ, example 150405H, Z-> timezone defined in bfe_util.TimeZoneMap